### PR TITLE
Remove obsolete static node pools (Legacy pools step 2/4)

### PR DESCRIPTION
**Pull request by linki**
Step 2/4 to remove legacy node pools:
* make CLM tolerate missing legacy node pool stack and userdata files (https://github.com/zalando-incubator/cluster-lifecycle-manager/pull/49)
* update stack to remove legacy resources and set "empty" default values for to-be-removed stack parameters (https://github.com/zalando-incubator/kubernetes-on-aws/pull/1091) <==
* stop specifying stack parameters in CLM. Default stack values from step 2 will take over. (https://github.com/zalando-incubator/cluster-lifecycle-manager/pull/46)
* finally remove unused stack parameters from stack config (https://github.com/zalando-incubator/kubernetes-on-aws/pull/1121)

Purpose of this PR:
* removes obsolete resources but keeps all stack parameters so that CLM will successfully invoke senza. It also sets defaults to each obsolete parameter so the final version of CLM can leave them out.

### CODE REVIEW

File: cluster/senza-definition.yaml [hunk 1]
```diff
@@ -921,8 +796,3 @@ Outputs:
       Ref: WorkerIAMRole
     Export:
       Name: "{{ Arguments.ClusterID }}:worker-iam-role"
-  MasterLoadBalancerSecurityGroup:
-    Value:
-      Ref: MasterLoadBalancerSecurityGroup
-    Export:
-      Name: "{{ Arguments.ClusterID }}:master-load-balancer-security-group"
```
Comment (mikkeloscar) [inline cluster/senza-definition.yaml:928]: This we still need :)

Comment (linki) [reply cluster/senza-definition.yaml:928]: I removed it because I couldn't find a place where it's referenced 😕

Comment (mikkeloscar) [reply cluster/senza-definition.yaml:928]: @linki true, then it's not needed as an export :+1:

**Comment by linki** [comment]
👍

**Comment by aermakov-zalando** [comment]
👍

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**